src/notation/internal/notationinteraction.cpp

```diff
@@ -1994,7 +1995,7 @@ void NotationInteraction::doAddSlur(const mu::engraving::Slur* slurTemplate)
             secondChordRest = mu::engraving::nextChordRest(firstChordRest);
         }
 
-        if (firstChordRest) {
+        if (firstChordRest && !(firstChordRest->isTrillCueNote() || secondChordRest->isTrillCueNote())) {
```

Comment:
I'm not sure but it might be wise to add a null check for `secondChordRest` here...
```suggestion
        if (firstChordRest && !(firstChordRest->isTrillCueNote() || (secondChordRest && secondChordRest->isTrillCueNote()))) {
```
(Inside `doAddSlur`, it seems not bad if `secondChordRest` is null, but for performing this check itself, it does matter of course.)

Reply:
Just tried it out: indeed a null check is necessary. (add a trill to the very last note of a score, and try to add a slur between the main note and the trill cue note)